boards/gardena/sgrm/board.yml

```
@@ -0,0 +1,9 @@
+# Copyright (c) 2024 GARDENA GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+
+board:
+  name: sgrm
```

```suggestion
  name: sgrm
  full_name: SiM3U radio module
```

But it looks as if "Smart Garden Radio Module SiM3U" might be better as it expands/explains the sgrm identifier (which tbh I'm not sure wouldn't be better with sim3u in it?)

---

No other flavors exists. Went with "smart garden radio module".

---

Not convinced about the all lower case but I take it that it might not be clear to folks what the purpose of this property is and that capitalization matters and will make sure to update the board porting template.
https://builds.zephyrproject.io/zephyr/pr/79401/docs/boards/index.html#vendor=gardena

---

Adapted as suggested. Not just here, but everywhere:

```console
$ rg -i "Smart Garden Radio Module"
boards/gardena/sgrm/sgrm.yaml
6:name: GARDENA Smart Garden Radio Module

boards/gardena/sgrm/sgrm.dts
12:     model = "GARDENA Smart Garden Radio Module";

boards/gardena/sgrm/board.yml
7:  full_name: Smart Garden Radio Module

boards/gardena/sgrm/doc/index.rst
3:GARDENA Smart Garden Radio Module
```

---

Perfect - thank you, sir!